torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -99,6 +100,8 @@ static void getMajorMinor(
   }
   major = dev_version.first;
   minor = dev_version.second;
+
+  return (major != nvrtc_version.first) || (minor != nvrtc_version.second);
```

Comment:
Are you comparing the right things here? `major` is like 8 for ampere, and nvrtc_version.first is 11?

Reply:
good catch.